examples/bare_metal_device/main.tf

```diff
@@ -13,6 +13,7 @@ resource "hivelocity_bare_metal_device" "demo" {
   location_name = "DAL1"
   hostname = "hivelocity.terraform.test"
   tags = ["hello", "world"]
+  script = "#cloud-config\npackage_update: true\npackages:\n - vim"
```

Comment:
I think we should go for a `file` example. 
Please check:
https://www.terraform.io/docs/language/functions/file.html

Reply:
my fault , I will improve this file example load, thanks Davinir